.github/workflows/main.yml

```
@@ -0,0 +1,61 @@
+name: HTTPStan
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+
+jobs:
+  tests:
+    name: HTTPStan tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+        python-version: [3.7, 3.8]
+      fail-fast: false
+    steps:
+      - name: Checkout github
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

```suggestion
      - name: Check out repository
        uses: actions/checkout@v2
```

- Check out is two words when used as a verb.
- I think ``fetch-depth: 0`` is incorrect. See the documentation: "Set fetch-depth: 0 to fetch all history for all branches and tags."

---

I think fetch-depth is also my legacy stuff from some repo with idk some problem.